test/checks/forms/fieldset.js

```diff
@@ -1,7 +1,9 @@
 describe('fieldset', function () {
 	'use strict';
 	var fixture = document.getElementById('fixture');
+	var shadowSupport = axe.testUtils.shadowSupport;
```

Comment:
This variable needs `.v1` at the end of it, or the tests below need it. There are some test failures as a result